combat-trainer.lic

```diff
@@ -2487,6 +2492,14 @@ class TrainerProcess
       bput('meditate research earthquake', 'You clear ')
     when /^Barb Research Utility$/i
       bput('meditate research prediction', 'You clear ')
+    when /^Landslide$/i
+      if DRStats.mana >= @barb_buffs_inner_fire_threshold
+        bput('berserk landslide', 'A supernatural', 'But you are already', 'Your inner fire lacks')
+      end
+    when /^Avalanche$/i
+      if DRStats.mana >= @barb_buffs_inner_fire_threshold
+        bput('berserk avalanche', 'The momentus rage', 'But you are already', 'Your inner fire lacks')
+      end
```

Comment:
💡  You may be able to reuse common-arcana helper methods here

```suggestion
    when /^Landslide$/i
      if DRStats.mana >= @barb_buffs_inner_fire_threshold
        DRCA. activate_barb_buff?('Landslide')
      end
    when /^Avalanche$/i
      if DRStats.mana >= @barb_buffs_inner_fire_threshold
        DRCA. activate_barb_buff?('Avalanche')
      end
```

Reply:
Yeah, why not? I think the names here would need to be updated to reflect adding "Berserk ..." in the previous committed suggestion. I'll make that happen and test the updated version